build.sh

```diff
@@ -107,25 +109,25 @@ function init_options() {
 function connectors() {
   pushd connectors
   # We are ALWAYS clean this project, because build fails otherwise: https://github.com/syndesisio/connectors/issues/93
-  mvn clean install $MAVEN_OPTS
+  "${BASEDIR}/mvnw" clean install $MAVEN_OPTS
```

Comment:
you could just use a `${MAVEN_CMD}` here which people might even be able to override if they want a different maven. 

but ok like this, too.